### ISSUE TITLE
Inconsistent button order between editor and panel

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#230122]
* open an editor (no tabs)
* open chat view in secondary sidebad
* the overflow (...) and close (X) commands are swapped

<img width="1069" alt="Screenshot 2024-09-30 at 15 56 31" src="https://github.com/user-attachments/assets/40817e6b-e6c5-45d8-b146-d09e2c6c8b7d">

**Comment by benibenj** [comment]
I agree, it also annoys me a little as it is inconsistent, these are two different concepts with two different behaviours. For the sidebar and the panel, the `x` means "close this container". In the case of the editor it means "close this view which is inside the container".

We have this already for a long time in the panel. Everything to the right of the `...` is a global action and acts on the entire container and can not be hidden. Everything to the left and including the `...` acts on the view inside the container and can be hidden. <img width="263" alt="image" src="https://github.com/user-attachments/assets/9d2abab4-d5e0-4b2a-b94a-fe8d30c6eff8">
